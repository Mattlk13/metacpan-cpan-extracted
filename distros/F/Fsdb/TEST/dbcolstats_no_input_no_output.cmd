prog='dbcolstats'
args='--no-output-on-no-input absdiff'
cmp='diff -c '
in=TEST/dbcolstats_no_input.in
