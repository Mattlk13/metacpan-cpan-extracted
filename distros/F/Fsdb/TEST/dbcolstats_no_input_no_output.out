#fsdb mean stddev pct_rsd conf_range conf_low conf_high conf_pct sum sum_squared min max n
# empty input
#   | dbcolstats --no-output-on-no-input absdiff
